operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ServerPod.java

```diff
@@ -213,6 +214,18 @@
       + "See `kubectl explain pods.spec.containers.volumeMounts`.")
   private final List<V1VolumeMount> volumeMounts = new ArrayList<>();
 
+  /**
+   * The common mount.
+   *
+   */
+  @Description("Use a common mount to automatically include directory content from additional images. "
+          + "This is a useful alternative for including Model in Image model files, or other types of files, in a pod "
+          + "without requiring modifications to the pod's base image 'domain.spec.image'. "
+          + "This feature internally uses a Kubernetes emptyDir volume and Kubernetes init containers to share "
+          + "the files from the additional images with the pod.")
+  @Feature("CommonMounts")
```

Comment:
In DomainSpec, you had @Feature("CommonMount"). Here you have the plural.

Reply:
I have changed the DomainSpec to use plural @Feature("Common Mounts") in 2865293 . Thanks,